File: eval/cases-triage/case-001-ready-clear-bug/src/main/java/com/example/user/UserProfile.java
package com.example.user;

public class UserProfile {
    private String name;
    private String email;
    private String phoneNumber;

    public UserProfile(String name, String email, String phoneNumber) {
        this.name = name;
        this.email = email;
        this.phoneNumber = phoneNumber;
    }

    public String getName() {
        return name;
    }

    public String getEmail() {
        return email;
    }

    public String getPhoneNumber() {
        return phoneNumber;
    }

    public String formatPhoneNumber() {
        if (phoneNumber.isEmpty()) {
            return "N/A";
        }

        String cleaned = phoneNumber.replaceAll("[^0-9]", "");
        if (cleaned.length() == 10) {
            return String.format("(%s) %s-%s",
                cleaned.substring(0, 3),
                cleaned.substring(3, 6),
                cleaned.substring(6, 10));
        }

        return phoneNumber;
    }

    public String getDisplayName() {
        return name + " (" + email + ")";
    }

    public String getContactInfo() {
        return "Phone: " + formatPhoneNumber();
    }
}

File: eval/cases/case-001-simple-null-pointer-fix/src/user_profile.py
"""User profile management module."""

import re


class UserProfile:
    """Represents a user profile with contact information."""

    def __init__(self, name: str, email: str, phone_number: str | None = None):
        self.name = name
        self.email = email
        self.phone_number = phone_number

    def format_phone_number(self) -> str:
        """Format the phone number for display.

        Returns:
            Formatted phone number or 'N/A' if empty.
        """
        if self.phone_number.strip() == "":
            return "N/A"

        cleaned = re.sub(r"[^0-9]", "", self.phone_number)
        if len(cleaned) == 10:
            return f"({cleaned[:3]}) {cleaned[3:6]}-{cleaned[6:10]}"

        return self.phone_number

    def get_display_name(self) -> str:
        """Return formatted display name."""
        return f"{self.name} ({self.email})"

    def get_contact_info(self) -> str:
        """Return contact info string including phone."""
        return f"Phone: {self.format_phone_number()}"
